crates/bevy_ui/src/focus.rs

```diff
@@ -127,8 +125,9 @@ impl Default for FocusPolicy {
 
 /// Contains entities whose Interaction should be set to None
 #[derive(Default)]
+#[cfg(feature = "bevy_render")]
 pub struct State {
-    entities_to_reset: SmallVec<[Entity; 1]>,
+    entities_to_reset: smallvec::SmallVec<[Entity; 1]>,
```

Comment:
Again, is there a reason for removing the `use smallvec::SmallVec;` just to use the FQN here instead? I personally prefer having the `use`-statement, since it makes it clear from the top of a file which crates are actually being used by this module.

Reply:
No, just conciseness. I'd be glad to change it. Thanks for the feedback.